packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipFormModal.tsx

```diff
@@ -81,7 +94,15 @@ export const UpdateMembershipFormModal = ({ onClose, onSubmit, member }: Props)
     mode: 'onChange',
   })
 
-  const [controllerAccount, rootAccount, handle] = form.watch(['controllerAccount', 'rootAccount', 'handle'])
+  const [controllerAccount, rootAccount, handle, isValidator, validatorAccountCandidate] = form.watch([
```

Comment:
I think the changes to this file (and the `types.ts` file) should be reverted and added in a followup PR because this flow might be more complex. E.g the validator accounts might be already partially bound to the existing membership so it's difficult to predict how many transactions will be needed. Also it will have to be tested so a new set of stories (I think "My Membership") have to be added.

WDYT ?